uptane-standard.md

```diff
@@ -587,17 +598,17 @@ The inventory database MUST record the following pieces of information:
     * The vehicle identifier the ECU is associated with
     * A public key
     * The format of the public key
-    * Whether the ECU is a primary or a secondary
+    * Identification of the ECU as a primary or a secondary
```

Comment:
I don't quite like the wording here. Two other items in the list use the term "identifier", but it means something completely different than "identification" here. Can I ask what the motivation for changing it was?

Reply:
Right, that's why I liked the simpler "Whether the ECU is a primary or a secondary": it states clearly and directly the informational content that's required, and I don't see how it can be misconstrued. "Identification of the ECU" sounds like it might be talking about some protocol for identification/authorization.

Reply:
But also, this is a great example of a place where we should *not* mandate a particular choice of how to represent the information. Why not have a primaries table and a secondaries table with foreign keys to the corresponding primary?

Reply:
I see what you mean now. Yes, I agree the earlier passage is more general.

Reply:
I made the change to maintain parallel structure. The first items in that bulleted list are actual objects--i.e specific pieces of information. They're nouns. Ending the list with something that is not a noun reads rather strange. We can change the word "Identification" if that could be confused as meaning something else but could we say "classification of the ECU as..." Obviously, clarity of meaning is our top priority, so I'll put the "whether " back for now but, it is a parallelism error as is.